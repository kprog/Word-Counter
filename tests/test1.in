esa vesa hesa. asuu helsingissÃ¤ esa.
