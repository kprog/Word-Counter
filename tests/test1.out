asuu 1
esa 2
helsingissä 1
hesa 1
vesa 1

virkkeitä: 2
3 sanan virkkeitä: 2
